### PR TITLE
Remove Scrollable1

### PR DESCRIPTION
**Pull request by abarth**
All the clients have migrated to Scrollable2.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: packages/flutter/test/physics/newton_test.dart [hunk 1]
```diff
@@ -282,8 +277,8 @@ void main() {
     expect(scroll.dx(0.0), closeTo(-7500.0, .0001));
 
     expect(scroll.isDone(0.025), false);
-    expect(scroll.x(0.025), closeTo(320.0, 1.0));
-    expect(scroll.dx(0.25), closeTo(-2982, 1.0));
+    expect(scroll.x(0.025), closeTo(317.0, 1.0));
+    expect(scroll.dx(0.25), closeTo(-4546, 1.0));
 
     expect(scroll.isDone(2.0), true);
     expect(scroll.x(2.0), 0.0);
```
Comment (Hixie) [inline packages/flutter/test/physics/newton_test.dart]: the changes in this file seem really sketchy but i assume you've checked that they make sense and aren't regression

File: packages/flutter/test/rendering/independent_layout_test.dart [hunk 2]
```diff
@@ -12,15 +12,18 @@ class TestLayout {
   TestLayout() {
     // viewport incoming constraints are tight 800x600
     // viewport is vertical by default
```
Comment (Hixie) [inline packages/flutter/test/rendering/independent_layout_test.dart]: this comment seems wrong now...

File: packages/flutter/test/rendering/offstage_test.dart [hunk 3]
```diff
@@ -14,17 +14,20 @@ void main() {
     bool painted = false;
     // viewport incoming constraints are tight 800x600
     // viewport is vertical by default
```
Comment (Hixie) [inline packages/flutter/test/rendering/offstage_test.dart]: ditto

File: packages/flutter/test/rendering/reattach_test.dart [hunk 4]
```diff
@@ -12,28 +12,31 @@ class TestTree {
   TestTree() {
     // viewport incoming constraints are tight 800x600
     // viewport is vertical by default
```
Comment (Hixie) [inline packages/flutter/test/rendering/reattach_test.dart]: ditto

File: packages/flutter/test/rendering/reattach_test.dart [hunk 5]
```diff
@@ -52,22 +55,25 @@ class TestCompositingBitsTree {
   TestCompositingBitsTree() {
     // viewport incoming constraints are tight 800x600
     // viewport is vertical by default
```
Comment (Hixie) [inline packages/flutter/test/rendering/reattach_test.dart]: ditto

**Comment by Hixie** [comment]
LGTM

File: packages/flutter/test/physics/newton_test.dart (hunk 1 above)
Comment (abarth) [inline packages/flutter/test/physics/newton_test.dart]: They're a result of the different drag coefficient.

**Pull request merged**